File: codingame/onboarding.scala
//usr/bin/env scala $0 $@;exit
import scala.util._
object Player{
	def main(args: Array[String]){
		while(true){
			val enemy1 = readLine // name of enemy 1
			val dist1 = readInt // distance to enemy 1
			val enemy2 = readLine // name of enemy 2
			val dist2 = readInt // distance to enemy 2
			println(if(dist1<dist2)enemy1 else enemy2)
		}
	}
}

File: codingame/onboarding.swift
//usr/bin/env swift $0 $@;exit
while true {
	let enemy1 = readLine()!
	let dist1 = Int(readLine()!)!
	let enemy2 = readLine()!
	let dist2 = Int(readLine()!)!

	print(dist1<dist2 ? enemy1 : enemy2)
}